net/pfSense-pkg-frr/files/usr/local/pkg/frr.inc

```diff
@@ -273,6 +274,16 @@ function frr_generate_config_rcfile() {
 		return null;
 	}
 
+	/* Get FRR daemon modules to append to daemon commands */
+	$zebradmodules = '';
+	$bgpdmodules = '';
+	$ospfdmodules = '';
+	$ospf6dmodules = '';
+	if ($config['installedpackages']['frr']['config'][0]['enablesnmpagentx']) { $zebradmodules .= '-M snmp'; }
+	if ($config['installedpackages']['frrbgp']['config'][0]['enablesnmpagentx']) { $bgpdmodules .= '-M snmp'; }
+	if ($config['installedpackages']['frrospfd']['config'][0]['enablesnmpagentx']) { $ospfdmodules .= '-M snmp'; }
+	if ($config['installedpackages']['frrospf6d']['config'][0]['enablesnmpagentx']) { $ospf6dmodules .= '-M snmp'; }
```

Comment:
These should either be done ternary style or ["K&R, BSD KNF variant style" formatting](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style), the way they are now is not a format we want in the code.